### ISSUE TITLE
Duplicate selector line 270 of content.css

### ISSUE THREAD

**Issue by jdelia** [wprig/wprig#216]
## Issue Overview
Version 2.0 - There appears to be a duplicate selector on line 270. It is an exact duplicate from line 269. 

```
/* stylelint-disable */
ul.wp-block-latest-posts.alignwide,
ul.wp-block-latest-posts.alignfull,
ul.wp-block-latest-posts.is-grid.alignwide,
ul.wp-block-latest-posts.is-grid.alignwide {
	padding: 0 1.5em;
}
/* stylelint-enable */
```

I noticed this on my install and stylelint was flagging an issue. 

## Possible Solution
If you either remove line 270 or replace **alignwide** with **alignfull** (which looks like that is what it should have been given the pattern of the first two rules ) - the warning is removed and you no longer need to disable stylelint for that block.

**Comment by mor10** [comment]
last line should be `alignfull`. Can you submit a PR?

**Comment by jdelia** [comment]
Yes, happy to. @mor10

**Comment by ataylorme** [comment]
Closed by #266
